Source/cursor.cpp

```diff
@@ -243,7 +243,7 @@ void FreeHalfSizeItemSprites()
 
 void DrawItem(const Item &item, const Surface &out, Point position, ClxSprite clx)
 {
-	const bool usable = item._iStatFlag;
+	const bool usable = MyPlayer == InspectPlayer ? item._iStatFlag : InspectPlayer->CanUseItem(item);

```

Comment:
Hm that kinda sounds like an issue

Reply:
that's why I made the comment :P

Reply:
Well, the stat flag is not being synced at all. It's not even part of the struct we use to sync item data.

https://github.com/diasurgical/devilutionX/blob/2296bf5625d76569405bff716ac929bec6b5ee69/Source/msg.h#L549-L566

I don't think it's an issue that it's not synced, but perhaps we should be calling `item.updateRequiredStatsCacheForPlayer(player)` on remote player's inventories. That said, it does feel a bit weird to me to use `_iStatFlag` on remote player inventories since we could just as easily want to see `item.updateRequiredStatsCacheForPlayer(*MyPlayer)` in some contexts, like what we do for shop menus. At least `InspectPlayer->CanUseItem(item)` is less ambiguous than simply reading `item._iStatFlag`.